### PR TITLE
Aggregation excel

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

**Comment by PavelBal** [comment]
Fixes #482

File: R/utilities-scenarios.R [hunk 1]
```diff
@@ -199,7 +199,7 @@ saveScenarioResults <- function(
         }
         # Save population
         if (!is.null(simulatedScenariosResults[[i]]$population) && all(!is.na(simulatedScenariosResults[[i]]$population))) {
-          ospsuite::exportPopulationToCSV(simulatedScenariosResults[[i]]$population$population,
+          ospsuite::exportPopulationToCSV(simulatedScenariosResults[[i]]$population,
```
Comment (PavelBal) [inline R/utilities-scenarios.R:202]: Was a bug from previous change.

**Comment by PavelBal** [comment]
Fixes #498

**Comment by PavelBal** [comment]
@Felixmil please review.

File: .gitignore [hunk 2]
```diff
@@ -54,3 +54,4 @@ tests/testthat/Rplots.pdf
 # {shinytest2}: Ignore new debug snapshots for `$expect_values()`
 *_.new.png
 .vscode
+/R/esqlabsR.code-workspace
```
Comment (Felixmil) [inline .gitignore:57]:
```suggestion
*.code-workspace
```

**Review by Felixmil — APPROVED** [review]

**Pull request merged**